server/protocols/amnezia.py

```
@@ -0,0 +1,59 @@
+
+
+class AmneziaProtocol(BaseProtocol):
+    PROTOCOL_ID = "amnezia"
+    PROTOCOL_NAME = "Amnezia"
+    DEFAULT_PORT = 51830
+
+    async def install(self) -> bool:
+        try:
+            await add_log("INFO", self.PROTOCOL_NAME, "Installing Amnezia dependencies...")
+            await self._apt_install("amneziawg-tools")
+            return True
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)